include/iceflow/producer-tlv.hpp

```diff
@@ -162,15 +164,16 @@ class ProducerTlv {
               m_dataManifestCountStream[m_stream]++;
             }
           }
-          // add the manifest to manifest storage
-          // store the manifest holding names of one result
-          // manifest prefix
-          // serialize the manifest and place it in the update storage
-          // create block from the manifest and store it
+          /**
+           * add the manifest to manifest storage
+           * store the manifest holding names of one result manifest prefix
+           * serialize the manifest and place it in the update storage
+           * create block from the manifest and store it
+           */
```

Comment:
Hmm, okay, sorry, I just noticed that the comments are not referring to the variable but describe the implementation. Maybe it would sense to create dedicated methods/functions for these operations then if they require a detailed explanation. For now, though, we can keep it as is.